source/sass/31-molecules/teaser/_teaser.scss

```diff
@@ -1,3 +1,9 @@
+@forward 'teaser--contact';
+@forward 'teaser--news';
```

Comment:
teaser--contact & teaser--news should be sub of this file, not the other way around.